tests/distributions/test_beta_binomial.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import torch
+
+from scvi.distributions import BetaBinomial
+
+
+def test_beta_binomial_distribution():
+    alpha = torch.ones(size=(2,))
+    beta = torch.ones_like(alpha)
+
+    num_trials = 100
+    total_count = torch.ones_like(alpha) * num_trials
+    x = torch.randint_like(total_count, high=num_trials)
```

Comment:
Would be great to also see a call to `log_prob` with an `x` outside of the distribution's support.

Reply:
Added a corresponding test